Scouting/src/main/java/ca/team2706/scouting/mcmergemanager/steamworks2017/dataObjects/CommentSingleton.java

```diff
@@ -0,0 +1,32 @@
+package ca.team2706.scouting.mcmergemanager.steamworks2017.dataObjects;
+
+import java.util.Set;
```

Comment:
I'm not totally sure I understand what this class is doing. Maybe it'll be more clear when I see what you have in mind for the file and server parts.